src/test/com/aca/homework/week7/car/tax/CarTypeTest.java

```diff
@@ -0,0 +1,11 @@
+package com.aca.homework.week7.car.tax;
+
+class CarTypeTest {
+    public static void main(String[] args) {
+        CarType electricCar = CarType.ELECTRIC;
+        CarType hybridCar = CarType.HYBRID;
+
+        System.out.println(electricCar.annualTex(2015));
```

Comment:
it is weird to have 2015 years old electric car )

Reply:
![Screenshot from 2022-02-23 17-23-14](https://user-images.githubusercontent.com/98351848/155328106-1b3184bb-04ce-42ca-bc95-4ce6e34adb2b.png)